### PR TITLE
react-elements | ChiaYu Lin

### PR DESCRIPTION
**Pull request by cyulin17**
react-elements-1

react-elements-2

react-elements-3

### CODE REVIEW

File: react-elements/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "webpack": "^5.56.1",
+    "webpack-cli": "^4.8.0"
```
Comment (scbowler) [inline react-elements/package.json:17]: `webpack` and `webpack-cli` should be installed as `devDependencies`. You can move a dependency to a dev dependency by running the following command: `npm run --save-dev package-name`, replace `package-name` with the actual name of the package.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just need to update your dependencies. Let me know if you have any questions.

File: react-elements/package.json (hunk 1 above)
Comment (cyulin17) [reply react-elements/package.json:19]: updated.

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**